packages/manager/apps/public-cloud/src/components/easter-eggs/modal/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "public_cloud_components_easter_eggs_modal_title": "Activation du Dark Mode (Alpha Version)",
+  "public_cloud_components_easter_eggs_modal_paragraph_1": "Bravo ! Vouz avez découvert la combinaison de touches vous permettant d’activer le Dark Mode sur votre espace client Public Cloud.",
+  "public_cloud_components_easter_eggs_modal_paragraph_2": "Vous pourrez revenir à tout moment au Light Mode en saisissant la même combinaison de touches.",
+  "public_cloud_components_easter_eggs_modal_question": "Êtes-vous sûr de vouloir activer le Dark Mode ?",
```

Comment:
Dark mode -> Mode sombre

Reply:
Didn't notice your comment before replying to: https://github.com/ovh/manager/pull/1832#discussion_r350106589.

I will add it to the next iteration.